frontend/src/metabase/nav/utils/admin-paths.ts

```diff
@@ -0,0 +1,57 @@
+import { User } from "metabase-types/api";
+import { PLUGIN_ADMIN_NAV_ITEMS } from "metabase/plugins";
+import { t } from "ttag";
+
+export const NAV_PERMISSION_GUARD: Record<
```

Comment:
I believe adding named properties could help in the future. e.g. 

```
interface PermissionGuard {
  "data-model": (user?: User) => boolean,
}
```

Reply:
I understand that you would have to add "audit" in that case, but I think that's OK.